bin/eslint-formatters-calypso/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "eslint-formatters-calypso",
+  "version": "0.0.3",
+  "main": "./index.js",
+  "bin": {
+    "eslines": "./index.js"
+  },
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint .",
+    "test": "./node_modules/.bin/tape tests/test*js | ./node_modules/.bin/faucet",
+    "test-cli": "./node_modules/.bin/tape tests/test-cli.js | ./node_modules/.bin/faucet",
+    "test-eslines": "./node_modules/.bin/tape tests/test-eslines.js | ./node_modules/.bin/faucet",
+    "test-exit-codes": "./node_modules/.bin/tape tests/test-exit-codes.js | ./node_modules/.bin/faucet",
+    "test-differ": "./node_modules/.bin/tape tests/test-differ.js | ./node_modules/.bin/faucet",
+    "test-parsing-errors": "./node_modules/.bin/tape tests/test-parsing-errors.js | ./node_modules/.bin/faucet"
+  },
+  "dependencies": {
```

Comment:
Do we need to run `npm install` in this directory in order for the command-line tool to work?